src/module/base/PromiseVerify.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+
+    async all(arr: any[]) {
```

Comment:
This method is written to manage tasks, it receives a list of promises, and if the result is equal to error, it displays the error to the user.